h2o-core/src/main/java/water/ExtensionManager.java

```diff
@@ -99,6 +102,9 @@ public void registerRestApiExtensions() {
     Log.info("Registered " + coreExtensions.size() + " core extensions in: " + registerCoreExtensionsMillis + "ms");
     Log.info("Registered H2O core extensions: " + Arrays.toString(getCoreExtensionNames()));
 
+    Log.info("Registered: " + getCoreExtensionNames().length + " listener extensions in: " + registerListenerExtensionsMillis + "ms");
```

Comment:
I would maybe log it just in debug for now (or when some listeners are actually enabled). The list will be empty most of the time, it might be confusing

Reply:
Good point

Reply:
Here was also bug that I was reporting `getCoreExtensionNames()` but should be `getListenerExtensionNames()`. Also fixed